website/www/site/content/en/blog/beam-2.39.0.md

```diff
@@ -0,0 +1,184 @@
+---
+title:  "Apache Beam 2.39.0"
+date:   2022-05-17 9:00:00 -0700
```

Comment:
I think we usually set the date when the release was approved/finalized, rather than when it was cut

Reply:
I'm aware, I'll change when it is being finalized.